code/app/src/main/java/com/example/the_tarlords/MainActivity.java

```diff
@@ -61,6 +63,8 @@ public class MainActivity extends AppCompatActivity implements OnMapReadyCallbac
     CollectionReference usersRef = db.collection("Users");
 
     public static User user;
+    // TODO : do not hardcode
+    public static Boolean isAdmin = true;
```

Comment:
Shouldn't need to hardcode this, it will be in firebase and we can just do user.getIsAdmin() once the user is initialized, and people can just add an isAdmin field to their user doc in firebase

Reply:
Ill comment out of main for the master , but if you're going to work on browse profiles then you need it. I added the firebase feild to my user but didn't know where to put user.getIsAdmin() without it returning null.